quotefault/mail.py

```diff
@@ -16,7 +16,7 @@ def send_report_email(reporter, quote):
     """
     Send email to eboard/rtp for a new report
     """
-    recipients = ["<eboard@csh.rit.edu>","<rtp@csh.rit.edu>"]
+    recipients = ["<eboard@csh.rit.edu>","<active-rtp@csh.rit.edu>"]
```

Comment:
Could you rebase? (or is github just not updated this yet)

Reply:
I merged jabbate/master to jabbate/random-quote, so I don't know how git responds to that. I'll rebase it.